src/lib/Containers/Conversation.tsx

```diff
@@ -293,6 +295,7 @@ export default createRefetchContainer(
           name
           initials
         }
+        initial_message
```

Comment:
we actually weren't using `initial_message` anywhere, so had to add it to the Relay fragment as well